sesman/chansrv/pulse/module-xrdp-sink.c

```diff
@@ -171,12 +171,12 @@ static void sink_update_requested_latency_cb(pa_sink *s) {
 
     u->block_usec = BLOCK_USEC;
     //u->block_usec = pa_sink_get_requested_latency_within_thread(s);
-    pa_log("1 block_usec %d", u->block_usec);
+    pa_log("1 block_usec %llu", (unsigned long long)u->block_usec);
```

Comment:
Please use space after cast here and elsewhere. I checked pulseaudio 10.0 sources, they use space too.